libs/chain/include/chain/transaction.hpp

```diff
@@ -174,5 +176,9 @@ class Transaction
   friend class TransactionSerializer;
 };
 
+using TransactionPtr = std::shared_ptr<Transaction>;
+
+using TransactionIndex = std::unordered_map<Digest, TransactionPtr, DigestHashAdapter>;
```

Comment:
```suggestion
using TransactionIndex = DigestMap<TransactionPtr>;
```

I  think the above it correct, in any case I think that it makes sense to use the helper type aliases

Reply:
Thanks, I've made an almost a full circle and forgot about it.